lib/matplotlib/tests/test_axes.py

```diff
@@ -6614,6 +6614,30 @@ def test_zoom_inset():
         axin1.get_position().get_points(), xx, rtol=1e-4)
 
 
+def test_inset_polar():
+    from matplotlib.projections.polar import PolarAxes
+
+    _, ax = plt.subplots()
+    axins = ax.inset_axes([0.2, 0.2, 0.3, 0.3], polar=True)
+    assert isinstance(axins, PolarAxes)
```

Comment:
This test seems good, but I do wonder if we actually need an image test here to make sure all the drawing gets through properly?  Otherwise this looks great.

Reply:
@jklymak thanks.  I have to admit I've never used matplotlib's own projections before, so will have to do some reading and have a think.  If you happen to know of any specific examples I could adapt, that would be great.

Reply:
@jklymak is something like [this](https://github.com/matplotlib/matplotlib/pull/22608/commits/452622505dd72a5ba6c6745e33772bcf63fc5ea5) sufficient?  From what I can tell, the zoom type plot won't work with polar, because the xlim/ylim don't correspond to a box.  But a user may have other reasons to want an inset plot.